scss/v2/components/_promo.scss

```diff
@@ -0,0 +1,93 @@
+.promo {
+    &__container {
+        display: block;
+
+        &:focus {
+            outline: 2px solid $sun-yellow;
+            outline-offset: 2px;
+            text-decoration: none;
+        }
+
+        &:hover {
+            text-decoration: none;
+        }
+    }
+
+    &__background {
+        &--blue-gradient {
+            background-image: linear-gradient(to bottom, $night-blue, $ocean-blue);
+            color: $white;
+        }
+
+        &--plum-gradient {
+            background-image: linear-gradient(to right, $indigo-blue, $plum-purple);
+            color: $white;
+        }
+    }
+
+    &__body {
+        display: flex;
+        height: 100%;
+        align-items: center;
+        justify-content: space-between;
+
+        &:before,
+        &:after {
+          content: '';
+          display: block;
+        }
+
+        &--column {
+            display: flex;
+            flex-direction: column;
+            justify-content: center;
+            align-items: flex-start;
+        }
+    }
+
+    &__copy {
+        @include breakpoint(sm) {
+            flex-basis: 75%;
+        }
+
+        @include breakpoint(md) {
+            flex-basis: 70%;
+        }
+        
+        @include breakpoint(lg) {
+            flex-basis: auto;
+        }
+    }
+
+    &__heading {
+        @include breakpoint(sm) {
```

Comment:
Not going say this needs to change, again just preference rather than having a specification for each breakpoint type having a default then using 2 saves a few lines of code - but not like it will save the environment :trollface: , likewise in __copy